package.json

```diff
@@ -6,32 +6,48 @@
   "engines": {
     "node": ">=18.19.0"
   },
+  "author": "Temitope Ayoku & SAS HOF Team",
   "scripts": {
     "start": "node server.js",
     "start:dev": "hof-build watch --env",
     "test": "NODE_ENV=test yarn run test:lint && yarn run test:unit",
     "test:lint": "eslint . --config ./node_modules/eslint-config-hof/default.js",
     "build": "hof-build",
+    "generate:reports": "node ./services/reports/generate_reports.js",
+    "generate:local:reports": "node -r dotenv/config ./services/reports/generate_reports.js",
+    "test:unit": "nyc _mocha \"test/_unit/**/*.spec.js\"",
+    "test:snyk": "snyk config set api=SNYK_TOKEN && snyk test",
     "postinstall": "hof-build"
   },
   "dependencies": {
     "accessible-autocomplete": "^2.0.4",
-    "axios": "^1.6.5",
-    "bl": "^6.0.10",
+    "aws-sdk": "^2.1333.0",
+    "axios": "^1.6.0",
+    "bl": "^6.0.1",
+    "bottleneck": "^2.19.5",
     "busboy": "^1.6.0",
     "bytes": "^3.1.2",
-    "dotenv": "^16.3.1",
-    "eslint-config-hof": "^1.3.1",
+    "csv-parse": "^5.3.8",
     "hof": "~20.4.0",
```

Comment:
most recent stable hof version is 20.5.0

Reply:
done